server/src/models/itemModel.js

```diff
@@ -0,0 +1,18 @@
+import { initDBConnection } from "../../db/db-connection.js";
+
+let db;
+
+export function initDb() {
+  db = initDBConnection();
+}
+
+export const itemModel = async (id) => {
```

Comment:
I'm assuming you'll have more queries in here so it would be good to clarify the names a bit more:

```suggestion
export const itemsByCategory = async (id) => {
```